integration-cli/requirements_test.go

```diff
@@ -36,6 +36,14 @@ func DaemonIsLinux() bool {
 	return testEnv.DaemonInfo.OSType == "linux"
 }
 
+func E2E() bool {
+	return os.Getenv("DOCKER_E2E") != ""
+}
```

Comment:
Commented on all the uses of this, I think this should be replaced with something that describes the requirement (a prestine/clean engine with no other resources) instead of the environment in which it runs (E2E). That will make it a lot easier to maintain because the requirement is more obvious from the name.

I would suggest something like `TEST_IS_PRESTINE_ENGINE` or something like that for the environment variable.

Reply:
I don't think there are any tests remaining that require this so will not add it now.